automated/android/apk-automation/apk-automation.yaml

```diff
@@ -27,16 +27,13 @@ params:
     APK_DIR: "./apks"
     BASE_URL: "http://testdata.validation.linaro.org/apks/"
     # Specify url and token for publishing artifacts.
-    ARTIFACTORIAL_URL: "https://archive.validation.linaro.org/artifacts/team/qa/"
-    # For safety reasons, please set 'ARTIFACTORIAL_TOKEN' variable in job definition with
+    # For safety reasons, please set 'ARTIFACTORIAL_TOKEN' or
+    # 'SQUAD_ARCHIVE_SUBMIT_TOKEN' variable in job definition with
     # 'secrets' dictionary, and set job visibility to personal or group.
     # Refer to https://validation.linaro.org/static/docs/v2/publishing-artifacts.html
-    ARTIFACTORIAL_TOKEN: ""
     # The SQUAD url to be used to upload the result and log files.
     # see https://squad.readthedocs.io/en/latest/intro.html#submitting-results.
-    # SQUAD_ARCHIVE_SUBMIT_TOKEN is used for uploading authentication,
-    # and must be defined by the submitter as one profile managed token
-    SQUAD_UPLOAD_URL: ""
+    UPLOAD_URL: "https://archive.validation.linaro.org/artifacts/team/qa/"
```

Comment:
This is related to the interface change, it would be better to support all of them for a while and drop them after that period.
Like the current status, there are jobs using ARTIFACTORIAL_URL(old jobs not working at the moment) and SQUAD_UPLOAD_URL(work now), and then will submit jobs to support "UPLOAD_URL" if we have this PR merged right now, then the jobs with SQUAD_UPLOAD_URL will fail, and it needs to resubmit them again. if both SQUAD_UPLOAD_URL and UPLOAD_URL are supported here, then the current jobs will be able to run, and it's possible to submit new jobs with UPLOAD_URL as well. ARTIFACTORIAL_URL support will be the similar.